java-manta-client/pom.xml

```diff
@@ -234,31 +234,31 @@
                     <relocations>
                         <relocation>
                             <pattern>com.joyent.http</pattern>
-                            <shadedPattern>com.joyent.manta.http</shadedPattern>
```

Comment:
What's the rational to change the shaded pattern? I'll admit the original package name was suboptimal.

If you want to change it, you will need to look for the code points where we detect if we are using a shaded artifact or not. If I recall correctly, this is being done in the kryo module and it may be done elsewhere. In those places, there is a string that refers to the shaded package name.